src/main/java/com/julienviet/pgclient/impl/PostgresSQLConnection.java

```diff
@@ -84,12 +101,12 @@ public void close() {
 
   @Override
   public SQLConnection commit(Handler<AsyncResult<Void>> handler) {
-    throw new UnsupportedOperationException();
+    return this;
   }
 
   @Override
   public SQLConnection rollback(Handler<AsyncResult<Void>> handler) {
-    throw new UnsupportedOperationException();
+    return this;
```

Comment:
I think we should keep the UnsupportedOperationException to mark it's not implemented